user-onboarding/src/components/UserForm.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useEffect } from "react";
+import { withFormik, Form, Field } from "formik";
+import axios from "axios";
+import * as Yup from "yup";
+
+const UserForm = ({ errors, touched, values, status }) => {
+  const [userName, setUserName] = useState([]);
+
+  useEffect(() => {
+    status && setUserName(userName => [...userName, status]);
```

Comment:
I would recommended this be `if (status)` rather then `status &&` simply so it is more clear in what it is doing.